docs/cicd.md

```diff
@@ -0,0 +1,214 @@
+| ----------------------- | --------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------- |
+| DELIVERINO_PRIVATE_KEY  | Base64 encoded PEM private key                                                    | Password Manager                                                                                                                            | [Deliverino Settings](https://github.com/organizations/Integreat/settings/apps/deliverino) | [Deliverino](https://github.com/apps/deliverino)                                         |
+| MM_WEBHOOK              | URL which can be used to send notifications to our mattermost. Keep this private! | Mattermost server settings                                                                                                                  | https://chat.tuerantuer.org/hooks/...                                                      | [Mattermost Documentation](https://docs.mattermost.com/developer/webhooks-incoming.html) |
+
+### Android Variables
+
+| Variable                       | Description                                                                                                | Where do I get it from?                                                  | Example                                 | Reference                                                                                              |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | --------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| GOOGLE_SERVICE_ACCOUNT_JSON    | JSON for authentication in the Google Play Console as Release Manager. This should expire after two years. | Password Manager                                                         | {...}                                   | [Service Account Docu](https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=de) |
+| CREDENTIALS_GIT_REPOSITORY_URL | Git remote URL to the credentials repository which contains the Java Keystore                              | Ask the team about this secret repository                                | git@github.com:User/credentials.git     | -                                                                                                      |
```

Comment:
CREDENTIALS_GIT_REPOSITORY_URL does not only contain the android keystore, but also the ios certificates, so this should be moved to the first table with the shared constants.